src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -29,6 +29,9 @@
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.lang3.ObjectUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.http.NameValuePair;
```

Comment:
We don't use Apache httpclient in our application i.e. it is not declared as a dependency in pom.xml, it's available as a transitive dependency only.
So let's either use other classes from the libraries declared as dependencies or explicitly declare Apache httpclient as dependency please.